### PR TITLE
Improve config loading and check termination

### LINKED ISSUE #1: Read configuration from URL / file
In order not to have to write a file to all cluster machines allow specifying an URL (`file:///...` / `https://...`) to fetch the configuration from.

### PR DESCRIPTION
**Pull request by Luzifer**
This PR introduces three new features:
- the ability to fetch the check configuration from an URL instead of a local file
- the ability to kill long running checks before they are started again to prevent spawning duplicate checks
- the ability to configure interval of config reloading and check execution

**references**
- fixes #1

### CODE REVIEW

**Comment by Luzifer** [comment]
Manually tested:

``` bash
# curl https://gist.githubusercontent.com/[...]/checks.yml
---

root_free_inodes:
  name: Ensure there are at least 30% free inodes on /
  command: test $(df -i | grep "/$" | xargs | cut -d ' ' -f 5 | sed "s/%//") -lt 70

longsleep:
  name: Sleep long
  command: sleep 120

# ./elb-instance-status -c 'https://gist.githubusercontent.com/[...]/checks.yml'
2016/07/22 12:11:23 Execution of check 'longsleep' was killed through context timeout.
^C
```

File: main.go [hunk 1]
```diff
@@ -59,11 +65,43 @@ func init() {
 }
 
 func loadChecks() error {
-	rawChecks, err := ioutil.ReadFile(cfg.CheckDefinitionsFile)
-	if err != nil {
-		return err
+	var (
+		rawChecks []byte
+		err       error
```
Comment (seiffert) [inline main.go]: Why do you declare `err` up here? As far as I can see, it is redeclared/shadowed in ln 73 and ln 86 and then used for the first time in ln 98, right?

Comment (Luzifer) [inline main.go]: fixed

**Comment by seiffert** [comment]
LGTM

**Pull request merged**